.github/workflows/tests.yml

```diff
@@ -13,17 +13,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - os: ${{ ((github.repository == 'commaai/opendbc') && ((github.event_name != 'pull_request') || (github.event.pull_request.head.repo.full_name == 'commaai/opendbc'))) && 'namespace-profile-amd64-8x16' || 'ubuntu-24.04' }}
-          - os: ${{ github.repository == 'commaai/opendbc' && 'namespace-profile-macos-8x14' || 'macos-latest' }}
+        os: ['namespace-profile-amd64-8x16','namespace-profile-macos-8x14']
     steps:
     - uses: commaai/timeout@v1
     - uses: actions/checkout@v4
     - run: ./test.sh
 
   safety_tests:
     name: safety
-    runs-on: ${{ ((github.repository == 'commaai/opendbc') && ((github.event_name != 'pull_request') || (github.event.pull_request.head.repo.full_name == 'commaai/opendbc'))) && 'namespace-profile-amd64-8x16' || 'ubuntu-latest' }}
+    runs-on: 'namespace-profile-amd64-8x16'
```

Comment:
forks don't have namespace (but fork PRs to upstream can)

Reply:
got it